### PR TITLE
remove all card design structure

### PR DESCRIPTION
**Pull request by Philip-Nunoo**
Remove all unnecessary selectors rendering aeIdentity as a card.

### CODE REVIEW

File: src/components/aeIdentity/aeIdentity.vue [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+    </div>
+    <div v-if="!collapsed" class="chunked-address">
+      <div v-for="chunk in chunkAddress" class="chunk">
+        <div v-for="data in chunk" class="row">
+          {{data}}
+        </div>
       </div>
     </div>
+    <div class="footer" v-if="hasSlot">
+      <slot></slot>
```
Comment (acdbaykal) [inline src/components/aeIdentity/aeIdentity.vue]: No slots pls!

File: src/components/aeIdentity/aeIdentity.scss [hunk 2]
```diff
@@ -33,35 +19,37 @@
...
   }
 
   .chunked-address {
-    display: flex;
-    flex-wrap: wrap;
     margin-top: 10px;
-    margin-left: -10px;
   }
 
   .chunked-address .chunk {
```
Comment (acdbaykal) [inline src/components/aeIdentity/aeIdentity.scss]: Why don't you just write .chunk? The parent element is not required to select these elements.

File: src/components/aeIdentity/aeIdentity.scss [hunk 3]
```diff
@@ -33,35 +19,37 @@
...
     font-weight: 500;
     font-style: normal;
-    color: $white;
     line-height: 1.53;
     letter-spacing: 3.1px;
     font-stretch: normal;
     text-align: left;
   }
+
+  .chunked-address .chunk .row {
```
Comment (acdbaykal) [inline src/components/aeIdentity/aeIdentity.scss]: Rename row into chunk-row and simply write .chunk-row (without the parents) pls.

File: src/components/aeIdentity/aeIdentity.scss [hunk 4]
```diff
@@ -1,23 +1,9 @@
...
-  box-shadow: 0 4px 8px 0 rgba(60, 60, 60, 0.1);
-  border-radius: 12px;
-  background-image: linear-gradient(122deg, #6f53a1, $aubergine);
-
-  &.active{
-    background-image: linear-gradient(119deg, #f22f70, #42255c);
-  }
+  color: $white;
 
   .footer {
```
Comment (acdbaykal) [inline src/components/aeIdentity/aeIdentity.scss]: Nesting is unnecessary. Pls move all the children out of .ae-identity.

File: src/components/aeIdentity/aeIdentity.vue [hunk 5]
```diff
@@ -1,32 +1,32 @@
...
-        <div v-for="chunk in chunkAddress" class="chunk">
-          {{chunk}}
+        <div :class="{balance: true}">
+          <span class="amount">{{amount}}</span>
+          <span class="currency-symbol">ETH</span>
         </div>
       </div>
-      <div class="footer" v-if="hasSlot">
-        <slot></slot>
+      <slot name="header-right"></slot>
```
[the remark this answers is not in the document]
Comment (Philip-Nunoo) [reply src/components/aeIdentity/aeIdentity.vue]: @acdbaykal do you mean the *footer* slot only?

the *header-right* and *header-left* slots are used for the purpose below
image

[the remark this answers is not in the document]
Comment (acdbaykal) [reply src/components/aeIdentity/aeIdentity.vue]: All slots should go. We will create another component called aeQuickId which inserts those buttons.

**Comment by acdbaykal** [comment]
@Philip-Nunoo I would like to merge https://github.com/aeternity/aepp-components/commits/update-ae-idenity-squash instead. It contains all the changes squashed into one commit. Is that ok for you?

**Comment by Philip-Nunoo** [comment]
@acdbaykal sure no problem

**Comment by acdbaykal** [comment]
Merged as stated above

**Pull request closed without merging**